packages/vkui/src/components/Popper/Popper.tsx

```diff
@@ -242,6 +253,10 @@ export const Popper = ({
           floatingDataY,
           sameWidth ? null : undefined,
         ),
+        ...(hideWhenReferenceHidden &&
+          hide?.referenceHidden && {
+            visibility: 'hidden',
+          }),
```

Comment:
кажется с включенным `hideMiddleware()` св-во `hide?.referenceHidden` не должно быть никогда `true` 🤔 

```suggestion
      ...(hide?.referenceHidden && {
          visibility: 'hidden',
        }),
```

Reply:
Да, выглядит как лишняя проверка, но это ещё и своего рода ранний return, что нету смысла проверять hide.
Убрал.
⚡ https://github.com/VKCOM/VKUI/pull/5690/commits/4d08d902a8e43cecfe00354917b024dc64083628